Shoot/README.md

```diff
@@ -15,4 +15,78 @@ $ cordova prepare ios
 $ open platforms/ios/HelloCordova.xcodeproj/
 ```
 
-Then follow the directions printed to tweak the native projects to be able to use the oauth plugin. See details instruction in [aerogear-oauth2-cordova](https://github.com/aerogear/aerogear-oauth2-cordova/blob/master/README.md#workaround-for-ios).
\ No newline at end of file
+Then follow the directions printed to tweak the native projects to be able to use the oauth plugin. See details instruction in [aerogear-oauth2-cordova](https://github.com/aerogear/aerogear-oauth2-cordova/blob/master/README.md#workaround-for-ios).
+
+## Facebook setup 
+
+### Step1: Setup facebook to be a facebook developer:
+
+- Go to [Facebook dev console](https://developers.facebook.com/products/login/)
+- Click Apps->Register as a Developer
+- enter password
+- accept policy
+- send confirmation code to SMS
+- once received enter code
+
+### Step2: Create a new app on facebook console
+
+- Click apps-> Create a new app
+- add display name: Shoot
+- deal with difficult catcha
+- configure Advanced setup:
+	- Native or desktop app? NO
+	- Client OAuth Login YES
+	- Embedded browser OAuth Login YES
+    
+### Step3: Configure Shoot app cordova client    
+
+Open app.js and update `clientId` and `clientSecret` with you `App ID` and `App Secret` and in case of iOS change `fbYYY` to `fb<App ID>` to the <project-name>-Info.plist:
```

Comment:
Sorry, it's probably obvious to the Cordova guys, but I have no idea where the app.js file is